`no such file or directory` error when copying the plugin file

Need to create folder if `~/.config/kn/plugins` is not exist. Otherwise an error will happen.

```
installing 'kn' plugin 'kn-source_kafka' from path '/Users/Daisy/go/src/github.com/daisy-ycguo/kn-source-kafka' to config path: /Users/Daisy/.config/kn
error copying plugin file to config directory: open /Users/Daisy/.config/kn/plugins/kn-source_kafka: no such file or directory
--- FAIL: TestSourceKafka (22.39s)
    source_kafka_test.go:54: assertion failed: error is not nil: open /Users/Daisy/.config/kn/plugins/kn-source_kafka: no such file or directory
    result_collector.go:101: === FAIL: =======================[[ERROR]]========================
        =================================================================
```

--- a/test/e2e/kn_plugin.go
+++ b/test/e2e/kn_plugin.go
@@ -59,9 +59,17 @@ func (kp *knPlugin) Install() error {
 		return err
 	}
 
+	pluginDir := filepath.Join(configDir, "plugins")
+	if !dirExists(pluginDir) {
+		err = os.MkdirAll(pluginDir, 0700)
+		if err != nil {
+			return err
+		}
+	}
+
 	fmt.Printf("installing 'kn' plugin '%s' from path '%s' to config path: %s\n", kp.pluginName, kp.pluginPath, configDir)
 
-	err = copyPluginFile(filepath.Join(kp.pluginPath, kp.pluginName), filepath.Join(configDir, "plugins", kp.pluginName))
+	err = copyPluginFile(filepath.Join(kp.pluginPath, kp.pluginName), filepath.Join(pluginDir, kp.pluginName))
 	if err != nil {
 		fmt.Printf("error copying plugin file to config directory: %s\n", err.Error())
 		return err

